DESCRIPTION

```diff
@@ -49,7 +49,8 @@ Imports:
     RColorBrewer,
     ape,
     lazyeval,
-    ggplot2
+    ggplot2,
+    tidyr
```

Comment:
Why the extra import here of tidyr?

Reply:
It doesn't look like any new functions are imported in the NAMESPACE

Reply:
my mistake, it was when we wanted to have the output in "wide" format. I used `tidyr::pivot_wider` to make the change. As we decided to go the "long" format instead, `tidyr` can be removed. I'll update this